gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -832,5 +844,15 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  zone_of_origin_prostate:
+    $ref:
+      - "_terms.yaml#/zone_of_origin_prostate/common"
+    enum:
+      - Peripheral zone
+      - Transition zone
+      - Central zone
+      - Overlapping/multiple zones
```

Comment:
Is there somewhere else in the dictionary that has a "/"?  I'm not sure if this will cause problems.